### DISCUSSION TITLE
[Icon Request] ordered and unordered list

### DISCUSSION THREAD

**Discussion by deodad** [tailwindlabs/heroicons#393] [Ideas]
Thank you for these quality icons!

![2020-08-28_10-18_1](https://user-images.githubusercontent.com/1143517/91584172-e264d500-e917-11ea-8a2b-d2a99fc03887.png)
![2020-08-28_10-18](https://user-images.githubusercontent.com/1143517/91584174-e264d500-e917-11ea-89c5-6d8b14e15cd9.png)

**Comment by deodad** [comment]
And even more, full text editor toolbar:
Heading
Bold
Italic

**Comment by BillWatts** [comment]
In addition:

- Strikethrough
- Underline
- Link
- Attachment

**Comment by brielov** [comment]
In addition:

- Align left
- Align center
- Align right

**Comment by kerrinhardy** [comment]
Yes, I would love to be able to use heroicons in a custom styling of the Trix toolbar
